src/js/components/award/table/cells/FederalAccountTableHeaderCell.jsx

```diff
@@ -0,0 +1,138 @@
+
+/**
+ * FederalAccountTableHeaderCell.jsx
+ * Created by Kevin Li 2/25/17
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Icons from 'components/sharedComponents/icons/Icons';
+
+import tableMapping from 'dataMapping/contracts/transactionTable';
```

Comment:
Sort is not working because this is the wrong mapping, so `sort.field` gets set to `undefined`